sale_timesheet_purchase/controllers/main.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (C) 2019 Odoo S.A.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from ast import literal_eval
+
+from odoo import _, http
+
+from odoo.addons.sale_timesheet.controllers.main import SaleTimesheetController
+from odoo.addons.web.controllers.main import clean_action
+
+from odoo.http import request
+
+
+class SaleTimesheetControllerNew(SaleTimesheetController):
+    def _plan_get_stat_button(self, projects):
```

Comment:
This is overriding/replacing the original method.
You need to call super, and then enrich the returned `stat_buttons`, to add the Purchase Orders and Vendor Bills.
Needs fixing.

Reply:
Chandresh, still no super() call